selftests/functional/test_output.py

```diff
@@ -391,11 +379,11 @@ def test_redirect_output(self):
     @unittest.skipIf(perl_tap_parser_uncapable(),
                      "Uncapable of using Perl TAP::Parser library")
     def test_tap_parser(self):
-        perl_script = script.TemporaryScript("tap_parser.pl",
```

Comment:
With this change, `from avocado.utils import script` is not needed anymore.  BTW, feel free to squash the unittest commit and your changes where it makes sense.